network/v1/001-clusternetwork-crd.yaml

```diff
@@ -23,6 +23,10 @@ spec:
         only one object of this type, named "default", which is created by the SDN
         network plugin based on the master configuration when the cluster is brought
         up for the first time.
+      type: object
+      required:
+      - clusterNetworks
+      - serviceNetwork
```

Comment:
Pulled in from `make update` - all CRD changes are in a separate commit, if necessary I can drop.